server/index.js

```diff
@@ -10,26 +10,8 @@ app.use(express.static(path.join(__dirname, '../client/dist')));
 app.get('/people/:org', (req, res) => {
   const company = req.params.org;
 
-  // company query is case sensitve
-  //   time permitting, should have set all company names as lowercase before seeding into db
-  //   then query lowercase name
-
-  // if a person updated their title multiple times,
-  // there are duplicate entries of that person, and we only want their most recent update
-  Person.aggregate([
```

Comment:
Interesting choice of moving this function out of the server file. What's your reasoning for doing so?

Reply:
I believe it's better practice to have the db query in the db file rather than my express server.